shells/android/javatests/arcs/crdt/parcelables/ParcelableCrdtSingletonTest.kt

```diff
@@ -0,0 +1,70 @@
+package arcs.crdt.parcelables
```

Comment:
Copyright comment?  (noticing they're missing on all of these files, actually..)

Reply:
Done.

I'll have a think about adding a lint check for this...